slsa_for_models/README.md

```diff
@@ -0,0 +1,85 @@
+# SLSA for Models
+
+To protect the supply chain of traditional software against tampering (like in
```

Comment:
I'd add a link to the blog post, since it's part of our work and this repo illustrates the concepts there.